core/modules/system/system.theme.inc

```diff
@@ -657,3 +657,61 @@ function _token_token_tree_format_row($token, array $token_info, $is_group = FAL
 
   return $row;
 }
+
+/**
+ * Prepares variables for Title Component block templates.
+ */
+function system_preprocess_title_components(&$variables) {
+  $settings = $variables['settings'];
+  $child_delta = $variables['child_delta'];
+  $variables += array(
+    'action_links' => NULL,
+    'tabs' => NULL,
+    'messages' => NULL,
+    'title' => NULL,
+  );
+
+  if ($child_delta == 'page_title' || $child_delta == 'title_combo') {
+    if ($settings['page_title_markup'] == 'none') {
+      $variables['page_title'] = backdrop_get_title();
```

Comment:
This is too early to call `backdrop_get_title()`. See if we can name/move this function to `layout_preprocess_title_components`.  Layout module has a really-heavy weight so that `backdrop_get_title()` can be called safely.